stdlib/map.gr

```diff
@@ -203,14 +270,37 @@ export let update = (key, fn, map) => {
   }
 }
 
+/**
+ * Returns the amount of key-value pairs stored within the map.
+ *
+ * @param map: The map to inspect
+ * @returns The amount of key-value pairs in the map
```

Comment:
```suggestion
 * Returns the number of key-value pairs stored within the map.
 *
 * @param map: The map to inspect
 * @returns The number of key-value pairs in the map
```

Reply:
@ospencer we generally don't use "number" because it is a type which will already be displayed in the table

Reply:
Yeah, that's fair. I don't know why amount sounds weird to me. Maybe count?

Reply:
We use `The count` in Stack.size documentation, so I can see that (and not be inconsistent). @spotandjake you'll need to update Set too.
```suggestion
 * Provides the count of key-value pairs stored within the map.
 *
 * @param map: The map to inspect
 * @returns The count of key-value pairs in the map
```

@ospencer what do you think about `Provides` instead of `Returns`? We use `Provides` in a couple other getters.

Reply:
I like provides!